blog/models.py

```diff
@@ -39,6 +39,7 @@ class Article(db.Document):
     author = db.ReferenceField(User)
     language = db.StringField(min_length=2, max_length=2, default='en')
     images = db.ListField()
+    is_published = db.BooleanField()
```

Comment:
I have a question about this.
I'll explain this with an example.
If I publish an article, then I unpublish this article, then I publish the article again, what will be the date of publication ? The first time I've published it or the last time ?